client/build.gradle.kts

```diff
@@ -32,8 +33,14 @@ import io.spine.internal.dependency.Spine
 
 plugins {
     // Add the Gradle plugin for bootstrapping projects built with Spine.
-    // See: https://github.com/SpineEventEngine/bootstrap
+    // See: https://github.com/SpineEventEngine/bootstrap.
     id("io.spine.tools.gradle.bootstrap").version("1.9.0")
+
+    // The ShadowJar plugin ID has been updated to `com.gradleup.shadow`, but since
+    // the required version is unavailable under this ID, the older plugin ID is used instead.
```

Comment:
You are being cryptic. Can you describe the problem in short and be specific, saying, e.g. the Gradle version X is required, but we have the Gradle version X-k.